### PR TITLE
Web clone

### PR DESCRIPTION
**Pull request by lily-coder**
This is milestone three of the clone of the Newsweek web page 
I have worked on the changes as requested by the reviewer in the previous pull request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -602,6 +602,577 @@ <h5 class="red-logo-h5"><strong>START YOUR DAY WITH OUR TOP FIVE ARTICLES</stron
...
+              <span class="heavy fs-24 icon font-roboto bg-orange">N</span>
+              <h4 class="fs-19 heavy font-roboto m-0">IN THE MAGAZINE</h4>
+            </div>
+          </div>
+
+          <section class="row border-top pt-4">
+            <div class="col-12 col-md-3">
+              <div class="top-img-div">
+                <a href="#">
+                  <img class="card-img" src="/img/newsweekpage.jpg" alt="">
```
Comment (sinansevgi) [inline index.html]:
```suggestion
                  <img class="card-img" src="./img/newsweekpage.jpg" alt="">
```
Without dot your image is not visible.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️ 

Good job! 
You fixed the issue with image alignments. But as previous TSE pointed out you still should fix below issues:

- [x] Your images in sections look stretched and higher than they should be.
Screenshot_2021-04-17 NewsWeek

- [x]  Newsletter subscription section doesn't look similar to original on mobile resolution

Yours
Screenshot_2021-04-17 NewsWeek(1)

Original
Screenshot_2021-04-17 Newsweek

- [x] First article of in the magazine section still doesn't looks as original

Screenshot_2021-04-17 NewsWeek(2)

----------------------------------
Optional 
- [x] Please remove unnecessary white space from project it will look unprofessional.
---------------------------------

"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, [you can request a second-opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s). "

-----
You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

File: index.html (hunk 1 above)
Comment (lily-coder) [reply index.html]: okay, corrected that

**Review by elisha2kyakpo1 — APPROVED** [review]
🎉💪✔**Congratulations @lily-coder, I'm happy to see all the changes requested being implemented**
I'm wishing you the best as you face bigger programs.

**[OPTIONAL] You can fix the images, they are not uniform, you can use background-image in CSS so that you have more control**
Cheers.💪👍

**Pull request merged**